tsconfig.build.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
```

Comment:
Created a separate `tsconfig.build.json` configuration file just for the build process to make sure we control the files that will be included in it.

(see comment on `tsconfig.json` file for extra information)